nextcord/components.py

```diff
@@ -313,9 +313,9 @@ def __init__(
         emoji: Optional[Union[str, Emoji, PartialEmoji]] = None,
         default: bool = False,
     ) -> None:
-        self.label = label
-        self.value = label if value is MISSING else value
-        self.description = description
+        self.label: str = label
+        self.value: str = label if value is MISSING else value
+        self.description: Optional[str] = description
```

Comment:
I don't think annotations are necessary here since these types can be inferred from the annotations in the arguments

Reply:
```
nextcord.ui.button.Button.label
  /.../nextcord/ui/button.py:error: Return type annotation is missing
```
pyright thought otherwise (`pyright --verifytypes nextcord`)

Reply:
I thought so too, pyright thought otherwise, maybe its so if the source type changes, the attribute type errors out, not sure.
```
  /.../nextcord/components.py:316:14 - error: Type is missing type annotation and could be inferred differently by type checkers
    Inferred type is "str"
```

Reply:
I find that strange, it's usually better to avoid writing the same thing twice since that can lead to more errors if it needs to change.

Reply:
Pyright is a much more powerful type checker than for example pyre or mypy, additionally those are fundamentally implemented differently which could cause them to infer those types differently.